src/components/sidebar/common/SectionResult.js

```diff
@@ -5,6 +5,7 @@ import ListItem from '@material-ui/core/ListItem';
 import ListItemText from '@material-ui/core/ListItemText';
 import Typography from '@material-ui/core/Typography';
 import { getFormattedClassSchedule, isUnscheduled } from 'util/time';
+import { withSnackbar, useSnackbar } from 'notistack';
```

Comment:
It looks like you didn't commit any changes to `package.lock` or `package-lock.json`, which will cause problems when anybody else tries to run the code after a fresh `npm install`. Make sure those changes are committed!

Reply:
In general, you want to commit your `package.lock` and `package-lock.json` changes whenever you install a new dependency or change the version of a existing dependency.